services/prompter/application-prompts.js

```diff
@@ -0,0 +1,54 @@
+const PromptUtils = require('./prompt-utils');
+
+class ApplicationPrompts extends PromptUtils {
+  constructor(program, envConfig, prompts, requests) {
```

Comment:
[Code Convention] The requests should be the first parameter as it is passed to the parent class